test/functional_ccs/apps/discover/_saved_queries_ccs.ts

```diff
@@ -0,0 +1,221 @@
+    await PageObjects.common.navigateToApp('discover');
+    await filterBar.addFilter('extension.raw', 'is one of', 'jpg');
+    await queryBar.setQuery('response:200');
+  };
+
+  describe('saved queries saved objects', function describeIndexTests() {
+    before(async function () {
+      log.debug('load kibana index with default index pattern');
+      await kibanaServer.savedObjects.clean({ types: ['search', 'index-pattern'] });
+
+      await kibanaServer.importExport.load(
+        'test/functional/fixtures/kbn_archiver/discover_ccs.json'
+      );
+      await kibanaServer.importExport.load(
+        'test/functional/fixtures/kbn_archiver/date_nested_ccs.json'
+      );
+      await esArchiver.load('test/functional/fixtures/es_archiver/date_nested');
+      await esArchiver.load('test/functional/fixtures/es_archiver/logstash_functional');
+
+      await kibanaServer.uiSettings.replace(defaultSettings);
+      log.debug('discover');
+      await PageObjects.common.navigateToApp('discover');
+      await PageObjects.timePicker.setDefaultAbsoluteRange();
+    });
+
+    after(async () => {
+      await kibanaServer.importExport.unload('test/functional/fixtures/kbn_archiver/discover_ccs');
+      await kibanaServer.importExport.unload(
+        'test/functional/fixtures/kbn_archiver/date_nested_ccs'
+      );
+      await esArchiver.unload('test/functional/fixtures/es_archiver/date_nested');
+      await esArchiver.unload('test/functional/fixtures/es_archiver/logstash_functional');
+      await PageObjects.common.unsetTime();
+    });
+
+    describe('saved query selection', () => {
+      before(async () => await setUpQueriesWithFilters());
+
+      it(`should unselect saved query when navigating to a 'new'`, async function () {
+        await savedQueryManagementComponent.saveNewQuery(
+          'test-unselect-saved-query',
+          'mock',
+          true,
+          true
+        );
+
+        await queryBar.submitQuery();
+
+        expect(await filterBar.hasFilter('extension.raw', 'jpg')).to.be(true);
+        expect(await queryBar.getQueryString()).to.eql('response:200');
+
+        await PageObjects.discover.clickNewSearchButton();
+
+        expect(await filterBar.hasFilter('extension.raw', 'jpg')).to.be(false);
+        expect(await queryBar.getQueryString()).to.eql('');
+
+        await PageObjects.discover.selectIndexPattern('remote:date-nested');
+
+        expect(await filterBar.hasFilter('extension.raw', 'jpg')).to.be(false);
+        expect(await queryBar.getQueryString()).to.eql('');
+
+        await PageObjects.discover.selectIndexPattern('remote:logstash-*');
+
+        expect(await filterBar.hasFilter('extension.raw', 'jpg')).to.be(false);
+        expect(await queryBar.getQueryString()).to.eql('');
+
+        // reset state
+        await savedQueryManagementComponent.deleteSavedQuery('test-unselect-saved-query');
+      });
+    });
+
+    describe('saved query management component functionality', function () {
```

Comment:
Dear @cuff-links @LeeDr , could you elaborate how this part is testing CCS functionality ? Because if not, I would suggest cleaning it up, since it's already tested in out non CCS functionals